layouts/partials/blog/sidebar.html

```diff
@@ -1,21 +1,30 @@
 <!--
     Blog sidebar, showing a list of tags, and a quick intro for context.
 -->
-<aside>
-    <h3 class="no-anchor">🍹 Pulumi Blog</h3>
-    <h4>Program the Cloud</h4>
-    <p>Create, deploy, and manage cloud infrastructure using your favorite language.</p>
-    <h5 class="no-anchor">Tags</h5>
-    <ul class="list-none p-0">
-        {{ $title := .Title }}
+<aside class="md:pr-8">
+    <h3 class="no-anchor">
+        <a href="{{relref . "/blog" }}">🍹 Pulumi Blog</a>
```

Comment:
🎉 Thanks! I was going to suggest making this a link.

Reply:
Yep. I was like, "I wanna click that." 😄